### PR TITLE
Feat/dist

### PR DESCRIPTION
**Pull request by tfbrito**
Added setuptools to allow package distribution

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    classifiers=[
+        'Environment :: Web Environment',
+        'Framework :: Django',
+        'Framework :: Django :: 1.10',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2.7',
+        'Topic :: Internet :: WWW/HTTP',
```
Comment (srtab) [inline setup.py]: Remove the Topics classifiers

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+setup(
+    name='django-mailings',
+    version='0.1',
+    packages=find_packages(),
+    include_package_data=True,
+    license='MIT',
+    description='Django module to easily send templated emails.',
+    long_description=README,
+    url='https://github.com/dipcode-software/',
```
Comment (srtab) [inline setup.py]: Change to repo address.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+from setuptools import find_packages, setup
+
+with open(os.path.join(os.path.dirname(__file__), 'README.md')) as readme:
+    README = readme.read()
+
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+setup(
+    name='django-mailings',
+    version='0.1',
```
Comment (srtab) [inline setup.py]: version compatible with semver: version='0.1.0'

**Comment by tfbrito** [comment]
Applied the changes suggested and improved the readme.

File: setup.py (hunk 1 above)
Comment (tfbrito) [inline setup.py]: Done.

File: setup.py (hunk 2 above)
Comment (tfbrito) [inline setup.py]: Done.

File: setup.py (hunk 3 above)
Comment (tfbrito) [inline setup.py]: Done.

**Pull request merged**